admiral/crd/dependencyproxies.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: dependencyproxies.admiral.io
+spec:
+  group: admiral.io
+  names:
+    kind: DependencyProxy
+    listKind: DependencyProxyList
+    plural: dependencyproxies
+    singular: dependencyproxy
```

Comment:
Can we add a shortName for the CRD, something like `dp`?

Reply:
yes, will do that